front-end/src/player/PlayerApp.tsx

```diff
@@ -116,10 +119,10 @@ export default function PlayerApp(props: PlayerAppProps) {
       return <PlayerIsSubject />;
     } else if (playerState === "seeing-answer-correct") {
       bottomButtons = false;
-      return <PlayerCorrect />;
+      return <PlayerCorrect pts={scoreDiff} />;
     } else if (playerState === "seeing-answer-incorrect") {
       bottomButtons = false;
-      return <PlayerIncorrect />;
+      return <PlayerIncorrect consolationPts={scoreDiff} />;
```

Comment:
this score diff will work only if the player does not refresh. otherwise, it will simply not show a score.